### PR TITLE
Correcting Plugin Configuration information [SKIP CI]

### PR DESCRIPTION
**Pull request by shuklanirdesh82**
`LogLevel` support is going to add through PR #1204 ... This PR handles the necessary document change. 

Documentation is part different branch other than master hence this separate PR.

### CODE REVIEW

File: jekyll-docs/configuration.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ The configuration for  Docker volume plugin require the driver type which can be
   <tbody>
     <tr>
       <td>driver</td>
-      <td>The name of Driver – vsphere for vSphere driver</td>
+      <td>The name of Driver – vsphere for vSphere driver. Possible values are vsphere/vmdk/photon.</td>
```
Comment (pshahzeb) [inline jekyll-docs/configuration.md]: vmdk is deprecated. Should we not mention it?

Comment (shuklanirdesh82) [inline jekyll-docs/configuration.md]: for customer who are still on vmdk and haven't upgraded their scripts ..for backward compatibility reason.

Comment (shuklanirdesh82) [inline jekyll-docs/configuration.md]: vSphere is anyway by default .. incase customer still want to use `vmdk` until their automated scripts to be changed to use `vsphere` driver.

**Comment by shuklanirdesh82** [comment]
Thanks @pshahzeb & @msterin !

**Pull request merged**